### PR TITLE
[Conflicts] Doctrine/annotations:^2.0

### PR DESCRIPTION
**Pull request by Rafikooo**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | 1.11                   |
| Bug fix?        | no                                                      |
| New feature?    | no                                                      |
| BC breaks?      | no                                                      |
| Deprecations?   | no |
| License         | MIT                                                          |

Reference: https://github.com/Sylius/Sylius/pull/14691

### CODE REVIEW

File: CONFLICTS.md [hunk 1]
```diff
@@ -99,6 +99,12 @@ In this section we keep track of the reasons, why some restrictions were added t
 
   References: https://github.com/PolishSymfonyCommunity/SymfonyMockerContainer/issues/20
 
+- `doctrine/annotations:^2.0`:
```
Comment (GSadee) [inline CONFLICTS.md]: As the conflict is added only to UiBundle, this note should be also added to that bundle.

File: src/Sylius/Bundle/UiBundle/CONFLICTS.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# CONFLICTS
+
+This document explains why certain conflicts were added to `composer.json` and
+references related issues.
+
+- `doctrine/annotations:^2.0`:
+
+  This version results in the following error:
+  `Error: Call to undefined method Doctrine\Common\Annotations\AnnotationRegistry::registerLoader()`
+   when running tests with `phpunit:8.5.x`
```
Comment (GSadee) [inline src/Sylius/Bundle/UiBundle/CONFLICTS.md:10]: Can you check whether the error will occur with version ^9.5? This is the version that we have in the composer of the main application.

Comment (Rafikooo) [reply src/Sylius/Bundle/UiBundle/CONFLICTS.md:10]: In each place in Sylius 1.11 we're using phpunit:^8.5. PHPUnit is bumped only in Sylius 1.13

Comment (GSadee) [reply src/Sylius/Bundle/UiBundle/CONFLICTS.md:10]: Yes, you're right, I looked wrong locally 🤦🏻‍♂️

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thanks, Rafał! :1st_place_medal:

**Pull request merged**